sw/device/tests/keymgr_key_derivation_test.c

```diff
@@ -2,64 +2,63 @@
 // Licensed under the Apache License, Version 2.0, see LICENSE for details.
 // SPDX-License-Identifier: Apache-2.0
 
-#include <stdbool.h>
 #include <stdint.h>
 
-#include "sw/device/lib/arch/device.h"
-#include "sw/device/lib/base/macros.h"
-#include "sw/device/lib/dif/dif_keymgr.h"
-#include "sw/device/lib/dif/dif_kmac.h"
-#include "sw/device/lib/dif/dif_otp_ctrl.h"
-#include "sw/device/lib/dif/dif_rstmgr.h"
-#include "sw/device/lib/runtime/hart.h"
-#include "sw/device/lib/runtime/log.h"
-#include "sw/device/lib/runtime/print.h"
 #include "sw/device/lib/testing/keymgr_testutils.h"
-#include "sw/device/lib/testing/kmac_testutils.h"
-#include "sw/device/lib/testing/rstmgr_testutils.h"
-#include "sw/device/lib/testing/test_framework/check.h"
 #include "sw/device/lib/testing/test_framework/ottf_main.h"
 
-#include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"
-#include "keymgr_regs.h"  // Generated.
-#include "kmac_regs.h"    // Generated.
-
```

Comment:
I think not all of these includes can be removed as a result of the changes in the test. Some could have already been removed before. If this is true, I would suggest factoring these removal out into a separate commit.

Reply:
Indeed, all the removed includes could have been removed before the changes of this commit. I think they were just copy-pasted from another file when the test was initially written because `keymgr_sideload_{aes,kmac,otbn}_test.c` have almost the same set unused imports. I split the removal into a separate commit.